vignettes/scoringutils.Rmd

```diff
@@ -61,21 +61,21 @@ set_forecast_unit(
 ```
 
 
-## Checking the input data
+## Constructing a forecast object
 
-The function `validate()` can be used to check the input data. It gives a summary of what `scoringutils` thinks you are trying to achieve. It infers the type of the prediction target, the prediction format, and the unit of a single forecasts, gives an overview of the number of unique values per column (helpful for spotting missing data) and returns warnings or errors.
+The function `as_forecast()` is be used to construct a forecast object and check the input data. It determines the forecast type, creates an object of the appropriate class (`forecast_binary`, `forecast_quantile` or `forecast_sample`), and validates the input data. Objects of class `forecast_*` have a print method that provides additional information. 
 
 ```{r}
 head(example_quantile)
 ```
 
 ```{r}
-validate(example_quantile)
+as_forecast(example_quantile)
```

Comment:
to match the recommended workflow this should be assigned and then printed to check its contents. You can then use this new object (i.e of type forecast) in all of the following.